tests/devices/test_default_qubit_tf.py

```diff
@@ -1355,7 +1362,8 @@ def cost(params):
 
         params = tf.Variable([theta, phi, lam], dtype=tf.float64)
 
-        with tf.GradientTape() as tape:
+        with tf.GradientTape(watch_accessed_variables=False) as tape:
+            tape.watch(params)
             tape.watch(params)
             res = cost(params)

```

Comment:
I assume the twice listed tape (
![image](https://user-images.githubusercontent.com/858615/140743765-34a330d0-4411-4eee-bc30-3dc9ce6ae970.png)
)
is just an artefact of how the diff handled the new addition.

Reply:
Yes! This is the case here.